conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
I'm not sure that this will give us the right re-rendering.

Have you tried something like re-rendering [patchelf]( https://github.com/conda-forge/pyelftools-feedstock ) on Windows or Mac (not Linux)? Does it change anything? For instance, does it delete `ci_support/run_docker_build.sh`?